partcad/src/partcad/runtime_python_conda.py

```
@@ -52,69 +64,104 @@
+            python_path = self.get_venv_python_path()
+            if os.path.exists(python_path):
+                try:
+                    p = subprocess.Popen(
+                        [python_path, "-c", "import sys; print(sys.version)"],
+                        stdout=subprocess.PIPE,
+                        stderr=subprocess.PIPE,
+                        shell=False,
+                        encoding="utf-8",
+                    )
+                    stdout, stderr = p.communicate()
+                    if not stderr is None and stderr.strip() != "":
```

Ruff (0.8.2)

Test for object identity should be `is not`

Convert to `is not`

(E714)